src/status_im/common/alert_banner/view.cljs

```diff
@@ -0,0 +1,57 @@
+(ns status-im.common.alert-banner.view
+  (:require [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.alert-banner.style :as style]
+            [utils.re-frame :as rf]))
+
+(def colors-map
+  {:alert
+   {:background-color colors/warning-50-opa-20
+    :text-color       colors/warning-60}
+   :error
+   {:background-color colors/danger-50-opa-20
+    :text-color       colors/danger-60}})
+
+(defn- banner
+  [{:keys [text type]} second-banner?]
```

Comment:
personally I prefer we just use the map for all params of components - involves a bit more destructuring etc but it's easier to read the component and see what's happening.

[banner error-banner false]
vs
```
[banner {:text 
                :second-banner? false
                :type :error (:text error-banner)
]
```